docs/readme.md

```diff
@@ -68,6 +69,20 @@ If the underlying data source does not allow such a fine-grained distinction, us
 }
 ```
 
+### `region`
+
+A region is a group of `station`s, for example a metropolitan area or a geographical or cultural region. A `station` can be part of multiple regions.
```

Comment:
I'd add backticks here around the "region" in the beginning and in the end of the sentence, to make sure the difference between the general concept of a region and our specific defined term is clear.

Reply:
Also, we should add more about the rationale behind `region`s. I think https://github.com/public-transport/friendly-public-transport-format/issues/7 contains useful pieces of text on that.

Reply:
E.g. something like this: 

> In many urban areas, there are several long-distance train & bus stations, all distinct but well-connected through local public transport. I makes sense to keep them as stations, because they may still have individual stops, but clustering them would enable more advanced routing information.